src/display/text_layer.js

```diff
@@ -46,6 +46,8 @@ var getDefaultSetting = displayDOMUtils.getDefaultSetting;
  *   initially be set to empty array.
  * @property {number} timeout - (optional) Delay in milliseconds before
  *   rendering of the text  runs occurs.
+ * @property {boolean} enhanceTextSelection - Whether to turn on the text
```

Comment:
Nit: similar to the `timeout` property above, let's add "(optional)" to this doc string.